docker/pypi/wmagent/init.sh

```diff
@@ -415,6 +421,10 @@ agent_tweakconfig() {
             GLOBAL_DBS_URL=https://cmsweb-testbed.cern.ch/dbs/int/global/DBSReader
             sed -i "s+DBSInterface.globalDBSUrl = 'https://cmsweb.cern.ch/dbs/prod/global/DBSReader'+DBSInterface.globalDBSUrl = '$GLOBAL_DBS_URL'+" $WMA_CONFIG_DIR/config.py
             sed -i "s+DBSInterface.DBSUrl = 'https://cmsweb.cern.ch/dbs/prod/global/DBSReader'+DBSInterface.DBSUrl = '$GLOBAL_DBS_URL'+" $WMA_CONFIG_DIR/config.py
+            rucio_host=http://cms-rucio.cern.ch
+            rucio_auth=https://cms-rucio-auth.cern.ch
+            sed -i "s+WorkQueueManager.rucioUrl = .*+WorkQueueManager.rucioUrl = $rucio_host+" $WMA_CONFIG_DIR/config.py
```

Comment:
This will break the configuration file, as you ditched out the single quotes making the url string.

Reply:
fixed
Here is the output of applying the lines to the config file:
```
(WMAgent-2.3.4rc3) [cmst1@vocms0260:current]$ sed -i "s+WorkQueueManager.rucioAuthUrl = .*+WorkQueueManager.rucioAuthUrl = '$rucio_auth'+" $WMA_CONFIG_DIR/config.py

(WMAgent-2.3.4rc3) [cmst1@vocms0260:current]$ sed -i "s+WorkQueueManager.rucioUrl = .*+WorkQueueManager.rucioUrl = '$rucio_host'+" $WMA_CONFIG_DIR/config.py

(WMAgent-2.3.4rc3) [cmst1@vocms0260:current]$ grep cms-rucio $WMA_CONFIG_DIR/config.py |grep WorkQueueManager
config.WorkQueueManager.rucioUrl = 'http://cms-rucio.cern.ch'
config.WorkQueueManager.rucioAuthUrl = 'http://cms-rucio-auth.cern.ch'

```

Reply:
Wonderful, thanks Todor!